corehq/motech/dhis2/views.py

```diff
@@ -89,9 +90,11 @@ def to_json(dataset_map):
 
 @require_POST
 @require_permission(Permissions.edit_motech)
```

Comment:
does `@require_pemission` imply a logged in user?
This also doesn't seem to do any domain checking, is that intentional?

Reply:
Oh, I am sorry. This change does not make sense in this PR. It should be in the next PR, "HTML UI". My mistake. I thought I'd caught all changes to views, and put them in there.

Yes, there will be a logged in user. This view serves the "Send Now" button for each DataSet Map in the `DataSetMapListView` page:
![image](https://user-images.githubusercontent.com/708421/118093415-b027d480-b3cd-11eb-975b-880f4028fc96.png)

When you say "This also doesn't seem to do any domain checking", can you say more? The function filters by domain when it selects `dataset_map`. The user's domain should be checked by the authentication middleware because this is a view. Does that affect your question?

Reply:
Thanks for the clarification, yes!